modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/util/PartitionlessLinks.java

```diff
@@ -29,25 +29,26 @@
 import java.nio.ByteBuffer;
 
 /**
- * Handling of <em>partitionless links</em>, that is, page memory links from which partition ID is removed.
+ * Auxiliary class for working with links and page IDs without partition ID.
  *
  * <p>They are used to save storage space in cases when we know the partition ID from the context.
  *
  * @see PageIdUtils#link(long, int)
+ * @see PageIdUtils#pageId(int, byte, int)
  */
 public class PartitionlessLinks {
     /** Number of bytes a partitionless link takes in storage. */
     public static final int PARTITIONLESS_LINK_SIZE_BYTES = 6;
 
     /**
-     * Reads a partitionless link from the memory.
+     * Reads a partitionless link or page ID from the memory.
      *
      * @param partitionId Partition ID.
      * @param pageAddr Page address.
      * @param offset Data offset.
-     * @return Partitionless link.
+     * @return Link or page ID with partition ID.
      */
-    public static long readPartitionlessLink(int partitionId, long pageAddr, int offset) {
+    public static long readPartitionless(int partitionId, long pageAddr, int offset) {
```

Comment:
I also asked to update comment in this method. Line `Page with index 0 is never a data page.` makes no sense now, it should be changed

Reply:
fix it